README.md

```diff
@@ -1,2 +0,0 @@
-# smm-deployment
-Repository to host docker compose script and helm chart for k8s deployemnt.
```

Comment:
I would keep this readme so the github has something to show? Just keep it as it is maybe?

Reply:
Oh I thought I created on in there. Weird. Good catch, thanks! Put it in there, but not much content in it.